src/ontology/fbcv-edit.obo

```diff
@@ -8881,6 +8885,16 @@ is_a: FBcv:0005005 ! reporter enzyme
 property_value: http://purl.org/dc/terms/contributor https://orcid.org/0000-0002-1373-1705
 property_value: http://purl.org/dc/terms/date "2024-04-12T10:18:38Z" xsd:dateTime
 
+[Term]
+id: FBcv:0007068
+name: gene product degradation tool
+namespace: experimental_tool_descriptor
+def: "Sequence that forms all or part of the gene product encoded by a transgenic locus or modified endogenous locus and which can be used to target a gene product for degradation in a defined manner." [FBC:GM]
+comment: Degradation tool may or may not be fused directly to the degredation target.
```

Comment:
spotted a typo - 'degredation' should be 'degradation'

Reply:
good spot! would have been picked up by spell check, but fixing [here](https://github.com/FlyBase/flybase-controlled-vocabulary/pull/227) anyway